CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
 cmake_minimum_required(VERSION 3.10)
 project(roboteam_utils VERSION 0.1 DESCRIPTION "utils for Roboteam Twente")
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
 SET(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake_modules")
 include(cotire)
 include(ccache)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 
+add_subdirectory(include/roboteam_utils/networking)
```

Comment:
why place this path in the include? Is has it's own include path, simply doing add_subdirectory(networking) would be cleaner IMO.

Reply:
Doens't work.

Reply:
No it does if you actually change the directory structure, which was my point lol.

Reply:
Don't you wanna just leave it this way and then change it when we re-design?

Reply:
Ok w/e. In the end I'll do a CMakeLists + include style pass and write a guide, I guess.